server/routes/api.js

```diff
@@ -79,7 +55,7 @@ module.exports = function Index({
       const agencyId = res.locals.user.activeCaseLoadId
 
       const results = await reportingService.getMostOftenInvolvedPrisoners(
-        systemToken(res.locals.user.username),
+        await systemToken(res.locals.user.username),
```

Comment:
A bit confused how this worked before without the `await`

Reply:
correct!, it didn't!